inference-engine/src/mkldnn_plugin/nodes/mkldnn_quantize_node.cpp

```diff
@@ -229,6 +234,9 @@ void MKLDNNQuantizeNode::init() {
             float ol = outputLowData[isOutputLowBroadcasted ? 0 : i];
             float oh = outputHighData[isOutputHighBroadcasted ? 0 : i];
 
+            if (ol == oh)
+                THROW_IE_EXCEPTION << "Quantize layer with name '" << getName() << "' has wrong output quantize ranges";
```

Comment:
Mathematically it is not prohibited `ol` to be equal to `oh` in binarization case.